assets/js/components/Dashboard/PaMessageForm/SelectStationsAndZones/SelectZonesPage.tsx

```diff
@@ -184,6 +184,80 @@ const SelectZonesPage = ({
     );
   };
 
+  const toggleIds = (ids: string[]) => {
+    onChange(
+      fp.difference(ids, value).length === 0
+        ? fp.difference(value, ids)
+        : fp.union(value, ids),
+    );
+  };
+
+  const renderRow = (place: Place, routeId: string) => {
```

Comment:
Any reason not to make this a `Row` component, accepting `{ place, routeId }` as props? I think it's a little unexpected/unconventional to have a function with a `ComponentType`-shaped return value that "isn't a component".

Reply:
Ah, I spaced and didn't realize this was defined inside of a component itself.